### PR TITLE
Catch the edge case when submitting a new project

### PR DESCRIPTION
**Pull request by singularo**
Errors printed by DrupalContentGenerate

* Change the default env domain name to be as it would be auto generated.
* Add phpcodesniffer auto config module.
* If there is no image stream yet:
  - don't try and get the value of the array
  - do give a warning when in the UI, content generate script is just back to silent now.

### CODE REVIEW

File: web/modules/custom/shepherd/shp_orchestration/src/Plugin/OrchestrationProvider/OpenShiftOrchestrationProvider.php [hunk 1]
```diff
@@ -273,24 +273,28 @@ public function createdEnvironment(
...
-        if ($image = $images['items'][0]['dockerImageReference'] ?? FALSE) {
-          $this->createCronJobs(
-            $deployment_name,
-            $cron_suspended,
-            $cron_jobs,
-            $image,
-            $cron_volumes,
-            $deploy_data
-          );
+    if (($image_stream = $this->client->getImageStream($sanitised_project_name))
```
Comment (pingers) [inline web/modules/custom/shepherd/shp_orchestration/src/Plugin/OrchestrationProvider/OpenShiftOrchestrationProvider.php]: Just do the assign first to clean up the if.

Comment (pingers) [reply web/modules/custom/shepherd/shp_orchestration/src/Plugin/OrchestrationProvider/OpenShiftOrchestrationProvider.php]: Then :ship:

Comment (singularo) [reply web/modules/custom/shepherd/shp_orchestration/src/Plugin/OrchestrationProvider/OpenShiftOrchestrationProvider.php]: Done

**Comment by singularo** [comment]
Actually, script does output the message.
```
🐳 simon@shepherd:/code$ robo dev:drupal-content-generate
 [Exec] Running bin/drush scr DrupalContentGenerate.php --uri=shepherd.172.17.0.1.xip.io
 [notice] Message: Build not yet complete, cron jobs cannot be created yet.

 [Exec] Done in 3.016s
```

**Pull request merged**